StyleGuide.mdown

```diff
@@ -0,0 +1,23 @@
+# Coding Styles
+
+## XSD Styles
```

Comment:
Though 1622 doesn't specifically have a style guide, I'd like to stick as close to their conventions as much as possible, even if it may be somewhat contrary to our own likings at times.